modules/virt-default-cluster-roles.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-additional-security-privileges-controller-and-launcher.adoc
+
+:_content-type: REFERENCE
+[id="default-cluster-roles-for-virt_{context}"]
+= Default cluster roles for {VirtProductName}
+
+{VirtProductName} includes a set of predefined cluster role objects that administrators can bind to users, granting them different levels of access to virtualization resources in the cluster.
```

Comment:
Here we should mention that the custom cluster roles use cluster role aggregation (https://docs.openshift.com/container-platform/4.12/authentication/using-rbac.html#cluster-role-aggregations_using-rbac) in order to automatically extend the default OpenSHift admin, edit, view, roles to include permissions to access the virt objects as well.

cc @davidvossel